components/inputs/input-time.js

```diff
@@ -141,6 +141,10 @@ class InputTime extends SkeletonMixin(FormElementMixin(LitElement)) {
 			 * @type {number}
 			 */
 			maxHeight: { type: Number, attribute: 'max-height' },
+			/**
+			 * Whether or not the dropdown is open
+			 */
+			opened: { type: Boolean, reflect: true },
```

Comment:
If this doesn't need to reflect for CSS reasons, probably best to not. I'm seeing non-zero performance implications of reflecting.

Reply:
Should this be updated? https://github.com/BrightspaceUI/guide/wiki/LitElement-Best-Practices-&-Gotchas#-do-reflect-properties-back-to-attributes

Reply:
Yeah that's a good call -- will update.